feature-tests/tests/resources/features/aisGET/InvokeApiGateWay-HappyPath.feature

```diff
@@ -3,115 +3,115 @@ Feature: Invoke-APIGateway-HappyPath.feature
     Scenario Outline: Happy Path - Get Request to /ais/userId - Returns Expected Data for <aisEventType>
         Given I send an <aisEventType> intervention message to the TxMA ingress SQS queue
         When I invoke the API to retrieve the intervention status of the user's account. With history <historyValue>
-        Then I expect the intervention to be <interventionType>, with the following state settings <blockedState>, <suspendedState>, <resetPassword> and <reproveIdentity>
+        Then I expect the intervention to be <interventionType>, with the following state settings <blockedState>, <suspendedState>, <resetPassword> and <reproveIdentity> with the <auditLevel> returned
         Examples:
-            | aisEventType              | historyValue | interventionType                                   | blockedState | suspendedState | resetPassword | reproveIdentity |
-            | pswResetRequired          | false        | AIS_FORCED_USER_PASSWORD_RESET                     | false        | true           | true          | false           |
-            | suspendNoAction           | false        | AIS_ACCOUNT_SUSPENDED                              | false        | true           | false         | false           |
-            | block                     | false        | AIS_ACCOUNT_BLOCKED                                | true         | false          | false         | false           |
-            | idResetRequired           | false        | AIS_FORCED_USER_IDENTITY_VERIFY                    | false        | true           | false         | true            |
-            | pswAndIdResetRequired     | false        | AIS_FORCED_USER_PASSWORD_RESET_AND_IDENTITY_VERIFY | false        | true           | true          | true            |
-            | unblock                   | false        | AIS_NO_INTERVENTION                                | false        | false          | false         | false           |
-            | userActionIdResetSuccess  | false        | AIS_NO_INTERVENTION                                | false        | false          | false         | false           |
-            | userActionPswResetSuccess | false        | AIS_NO_INTERVENTION                                | false        | false          | false         | false           |
-            | unSuspendAction           | false        | AIS_NO_INTERVENTION                                | false        | false          | false         | false           |
+            | aisEventType              | historyValue | interventionType                                   | blockedState | suspendedState | resetPassword | reproveIdentity | auditLevel |
+            | pswResetRequired          | false        | AIS_FORCED_USER_PASSWORD_RESET                     | false        | true           | true          | false           | standard   |
```

Comment:
Hi @liamquayle, can you please remove this audit level from here and just add validation bit in the step def file since its common for each event.